build.gradle

```diff
@@ -0,0 +1,59 @@
+plugins {
+    id 'java'
+    id 'jacoco'
+    id 'checkstyle'
+    id 'pmd'
+}
+
+group 'no.unit.nva'
+version '1.0-SNAPSHOT'
+
+repositories {
+    jcenter()
+    maven { url "https://jitpack.io" }
+}
+
+def jacksonVersion = '2.10.2'
+def jerseyVersion = '2.30'
+
```

Comment:
The tests do not actually run when gradle runs, since we are running junit vintage tests to some extent, you need: 

```
dependencies {
…

    testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.6.0'
    testCompileOnly 'junit:junit:4.12'
    testRuntimeOnly 'org.junit.vintage:junit-vintage-engine:5.1.0'
}

test {
    useJUnitPlatform()
    failFast = true
    finalizedBy jacocoTestReport
}

```